serverlessworkflow/modules/ROOT/pages/tooling/serverless-workflow-editor/swf-editor-vscode-extension.adoc

```diff
@@ -27,19 +27,19 @@ image::tooling/serverless-workflow-editor/swf-editor-vscode-extension-page.png[]
 .Procedure
 . You can install the VS Code extension for Serverless Workflow editor using one of the following methods:
 
-* *Go to Visual Studio Marketplace*: Go to link:{kogito_swf_editor_vscode_marketplace_url}[Serverless Workflow Editor] extension page in Visual Studio Marketplace and click *Install*.
+* *Go to Visual Studio Marketplace*: Go to link:{kogito_swf_editor_vscode_marketplace_url}[KIE Serverless Workflow Editor] extension page in Visual Studio Marketplace and click *Install*.
```

Comment:
Aligned with my comment in the other PR, maybe we can create a `{product_tooling_name}` instead. So we easily change it once we do the downstream branch.

Reply:
In the other PR I changed all occurences in Dev UI and tooling to use the `{product_name}` where applicable. Here I introduced the `{serverless_workflow_vscode_extension_name}` variable because this name is specific to the VS Code extension. Thanks.